lintlizard/__init__.py

```diff
@@ -37,17 +37,18 @@ def version_command(self) -> Command:
 
 
 TOOLS = [
+    # Python source formatters
+    CommandTool('mypy'),
     CommandTool('flake8', default_files=()),
     CommandTool(
         'isort', run_params=('-c',), fix_params=(), default_files=('.',)
     ),
-    CommandTool('mypy'),
     CommandTool(
-        'black',
-        run_params=('--check',),
-        fix_params=(),
-        default_files=('.',),
+        'black', run_params=('--check',), fix_params=(), default_files=('.',),
     ),
+
+    # Markdown source formatters
+    CommandTool('mdformat', run_params=('--check', '--number', '--wrap 60'))
```

Comment:
Since `mdformat` is not just a linter, but also works as autoformatter, specifying `fix_params=()` will enable auto-formatting with `lintlizard --fix`

Reply:
passing `fix_params=(`--check`,)` should address this right?